### DISCUSSION TITLE
Quality of life feature sugestion: Mr. Lapin should give two pet packs upon completing the second mission

### DISCUSSION THREAD

**Discussion by Proxiehunter** [cataclysmbn/Cataclysm-BN#2068] [General]
Encountered Mr. Lapin for the first time and completed both quests. Apparently he gives enough cattle fodder to befriend two rabbits, but only enough pet packs to transport _one_ requiring the player to either already have one on them or make two trips to transport a breading pair of rabbit back to base. It would save some tedium if he just gave two pet packs.

Alternatively a third quest could be added to _earn_ the second pet pack but I don't think just giving the player two would unbalance things.  

Any thoughts?

**Comment by scarf005** [comment]
resolved in #2085
